src/bin/scribble.ts

```diff
@@ -267,25 +264,14 @@ function instrumentFiles(
         assert(contents !== undefined, `Missing source for ${unit.absolutePath}`);
 
         for (const fun of unit.vFunctions) {
-            const annotations = getAnnotationsOrDie(fun, ctx.files, filterOptions);
-
-            if (annotations.length > 0) {
-                const annotation = annotations[0];
-
-                prettyError(
-                    "UnsupportedError",
-                    "Instrumenting free functions is not supported yet",
-                    unit,
-                    annotation.predicateFileLoc(contents),
-                    annotation.original
-                );
-            }
+            getAnnotationsOrDie(fun, ctx.files, filters);
```

Comment:
Should add a comment here to explain the logic:

// We call getAnnotationsOrDie to make sure there are no annotations on free functions

Reply:
Addressed in https://github.com/ConsenSys/scribble/pull/5/commits/3b1a65c0b0e464efe1aaada050bfe36bd5592678.